lntest/rpc/harness_rpc.go

```diff
@@ -73,6 +75,7 @@ func NewHarnessRPC(ctxt context.Context, t *testing.T, c *grpc.ClientConn,
 		ChainKit:         chainrpc.NewChainKitClient(c),
 		NeutrinoKit:      neutrinorpc.NewNeutrinoKitClient(c),
 		Peer:             peersrpc.NewPeersClient(c),
+		DevRPC:           devrpc.NewDevClient(c),
```

Comment:
nit: not required anymore. But I guess doesn't hurt to leave it in either, so up to you.

Reply:
removing